connection.go

```diff
@@ -63,12 +64,16 @@ func (mc *mysqlConn) handleParams() (err error) {
 				return
 			}
 
-		// System Vars
+		// Other system vars
 		default:
-			err = mc.exec("SET " + param + "=" + val + "")
-			if err != nil {
-				return
-			}
+			params = append(params, param+"="+val)
```

Comment:
nitpick (not a must): This allocates a new string for every param. Maybe using [`strings.Builder`](https://golang.org/pkg/strings/#Builder) (and adding the `,` here) is the better approach?

Reply:
I know there are improvement in this area. For this important pull request I wanted to apply the minimum change required. The string concatenation problem was in the original code, so I didn't want to change that and divert the reviewers from the main issue.

Now that this structural change is merged (thanks!), I am now able to propose some further implementation improvements.